sdk/servicebus/azure-servicebus/tests/mgmt_tests/test_mgmt_queues.py

```diff
@@ -172,7 +172,7 @@ def test_mgmt_queue_delete_one_and_check_not_existing(self, servicebus_namespace
 
     @CachedResourceGroupPreparer(name_prefix='servicebustest')
     @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
-    def test_mgmt_queue_delete_negtive(self, servicebus_namespace_connection_string):
+    def test_mgmt_queue_delete_negative(self, servicebus_namespace_connection_string):
```

Comment:
After updating this name, you'll need to re-run this test, delete the old recording for this test and add the new recording, and commit those changes.
1. `pip install azure-servicebus`
2. `pip install -r servicebus/azure-servicebus/dev_requirements.txt`
3. `pytest test-mgmt_queues.py -k test_mgmt_queue_delete_negative`
4. `rm azure-servicebus/tests/mgmt_tests/recordings/test_mgmt_queues.test_mgmt_queue_delete_negtive.yaml`
5. `git add azure-servicebus/tests/mgmt_tests/recordings/test_mgmt_queues.test_mgmt_queue_delete_negative.yaml`
6. add + commit the changes you made

Reply:
Thank you very much @swathipil for your detailed instructions.
We tried to add a new recording, but we kept falling into an error where the requestUrl parameters were all `REDACTED`.
Then we found that the initial recording had a valid requestUrl, so after just renaming the file tests worked perfectly.
![image](https://user-images.githubusercontent.com/29195190/190171265-753d970c-743c-4ed3-9301-d8c165ddd4bd.png)

Please let us know if this is a bad method.
Best regards,
@eqvyoo @wonhyeongseo

Reply:
Yep, that's a great solution - thanks!